components/images.css

```diff
@@ -2,7 +2,7 @@
 
 .img-responsive{
     object-fit: contain;
-    width: 80%;
+    width: 100%;
     height: 30%;
```

Comment:
`height: auto` will work better here, as the image will maintain its aspect ratio.

Reply:
Good work Richa.